### DISCUSSION TITLE
Please suggest me best way to implement licensing for my Microsoft Access Database?

### DISCUSSION THREAD

**Discussion by ahmadkakarr** [keygen-sh/community#94] [General]
I am building a management system solution in Microsoft access to sell. i want to implement licensing in it using Visual basic. Please suggest the best method to do implement licensing in Microsoft Access. It would be very help full if you give code examples too in VB.

Thanks

**Comment by ezekg** [comment]
This is the only example for VB I have: https://gist.github.com/ezekg/ebcf312f62f01dc71902f0a333099137

I'm not super familiar with VB so I probably won't be too much help here, sorry.
